api-module-library/microsoft-teams/test/bot.test.js

```diff
@@ -42,7 +42,7 @@ describe(`${config.label} API Tests`, () => {
                 "serviceUrl": "https://smba.trafficmanager.net/amer/"
             };
             api.conversationReferences[ref.user.email] = ref;
-            const resp = await api.bot.sendProactive(ref.user.email, 'proactive message');
+            const resp = await api.sendProactive(ref.user.email, 'proactive message');
             // resp is undefined for now, even when message is sent
```

Comment:
Oh interesting. How would we go about updating the sent message? Find it in the DM conversation and then update? (thinking about changing a block based on user interaction)